api/proquest/scripts.py

```diff
@@ -14,6 +13,18 @@ def __init__(self, *args, **kwargs):
 
         self._logger = logging.getLogger(__name__)
 
+    @classmethod
+    def arg_parser(cls):
+        parser = OPDSImportScript.arg_parser()
+        parser.add_argument(
+            "--process-removals",
+            help="Remove from the Circulation Manager's catalog items that are no longer present in the ProQuest feed",
+            dest="process_removals",
+            action="store_true",
```

Comment:
The default value here will be false. If this is functionality you only want to activate once a week or so, then that's the right decision--you'll just have to add the script to the crontab twice.